planetary-social/nos
# Content flag send selection category

## Issues covered
 [#1499](https://github.com/planetary-social/nos/issues/1499)
Part 2 of [#1489](https://github.com/planetary-social/nos/issues/1489)

## Description
This PR includes the functionality to choose how to flag a content in the new moderation flow.

## How to test
1. Build the app
2. Click the side menu
3. Click the Settings on the side menu
4. Scroll down to turn on the toggle for new moderation flow
5. Go back to the Feed screen by clicking the feed tab at the bottom of the screen
6. Click on the 3 dots on one of the notes.
7. Select `Flag this content` option. 
8. Select an option from the categories displayed.
9. Please confirm that new categories on how to flag a content shows up under the first category.

## Screenshots/Video


https://github.com/user-attachments/assets/bf13beff-3c13-45ea-b464-01d7478f36f7
